hw2/02-reverse.html

```diff
@@ -14,17 +14,43 @@
     <main class="bg-light border rounded w-50 mx-auto mt-5 p-3">
       <h1 class="mt-2 mb-4">Reverse</h1>
       <div class="form-group w-50">
+        <!-- Input label -->
         <label for="input">Enter an 8-digit number: </label>
         <input type="number" class="form-control" id="input" required />
       </div>
       <div class="form-group mt-4">
+        <!-- Submit Button -->
         <input
           type="submit"
           class="btn btn-primary"
           value="Reverse"
           id="reverse"
+          onclick="getReversedResults()"
         />
       </div>
+      <div>
+        <br>
+        <text id="result"></text>
+      </div>
     </main>
   </body>
+    <script>
```

Comment:
I would move the JavaScript from the html file to the JavaScript file. This will make the code easier to read. Use let and const over var referenced from the Airbnb style guide.